internal/services/executors/transport/graphql/executor_resolver.go

```diff
@@ -39,6 +42,56 @@ func (e *ExecutorResolver) SrcCliVersion() string   { return e.executor.SrcCliVe
 func (e *ExecutorResolver) FirstSeenAt() DateTime   { return DateTime{e.executor.FirstSeenAt} }
 func (e *ExecutorResolver) LastSeenAt() DateTime    { return DateTime{e.executor.LastSeenAt} }
 
+func (e *ExecutorResolver) IsOutdated() (bool, error) {
+	ev := e.executor.ExecutorVersion
+	return isExecutorOutdated(ev, e.Active())
```

Comment:
Maybe should an inactive executor just early return here?

```suggestion
    if !e.Active() {
        return false, nil
    }
	return isExecutorOutdated(ev)
```

It feels unrelated to the version check and at first I wondered if there was a different relationship between the version and whether or not the executor is active. 😅